docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -1,40 +1,30 @@
 [id="configuring-a-ceph-backend_{context}"]
 
-= Configuring a Ceph backend
+= Configuring a {Ceph} back end
 
-If the original deployment uses a Ceph storage backend for any service
-(e.g. {image_service_first_ref}, {block_storage_first_ref}, {compute_service_first_ref}, {rhos_component_storage_file_first_ref}), the same backend must be used in the
-adopted deployment and custom resources (CRs) must be configured accordingly.
-
-If you use {rhos_component_storage_file_first_ref}, on {OpenStackPreviousInstaller} environments, the CephFS driver in {rhos_component_storage_file} is configured to use
-its own keypair. For convenience, modify the `openstack` user so that you
-can use it across all {rhos_prev_long} services.
-
-Using the same user across the services serves two purposes:
-
-* The capabilities of the user required to interact with {rhos_component_storage_file}
-became far simpler and hence, more became more secure with {rhos_acro} {rhos_curr_ver}.
-* It is simpler to create a common ceph secret (keyring and ceph config
-file) and propagate the secret to all services that need it.
+If your {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} deployment uses a {Ceph} back end for any service, such as {image_service_first_ref}, {block_storage_first_ref}, {compute_service_first_ref}, or {rhos_component_storage_file_first_ref}, you must configure the custom resources (CRs) to use the same back end in the {rhos_long} {rhos_curr_ver} deployment.
 
 [TIP]
-To run `ceph` commands, you must use SSH to connect to a Ceph
-storage node and run `sudo cephadm shell`. This brings up a ceph orchestrator
-container that allows you to run administrative commands against the ceph
-cluster. If you deployed the ceph cluster by using {OpenStackPreviousInstaller}, you may launch the `cephadm` shell from an {OpenStackShort} controller node.
+If you use {rhos_component_storage_file} on {OpenStackPreviousInstaller} environments, the CephFS driver in {rhos_component_storage_file} is configured to use its own key pair. For convenience, modify the `openstack` user so that you
+can use it across all {OpenStackShort} services.
```

Comment:
Do we want to call out the reason of this choice? One important reason is that `Manila`, which is used to be configured with its own Ceph keyring in 17,  doesn't require more privileges anymore compared to other OpenStack services backed by Ceph, hence using a single, openstack, keyring is simpler.

Reply:
Yeah, in theory this is not a mandatory step for users that have reasons to keep the keyring separated. In practice, for greenfield, we support a single keyring for the openstack services, so keeping a different keyring might result in issues, and we don't even document that kind of use case, so this make it not really optional. I would remove that wording if it helps, and ask to make this change anyway.